.github/workflows/test.yml

```diff
@@ -29,4 +29,8 @@ jobs:
       - run: |
           pip install poetry && poetry install
       - run: |
-          poetry run pytest -v tests.py
+          poetry run pytest --cov=sansio_lsp_client --cov-report=xml:cov.xml -v tests.py
+      - uses: actions/upload-artifact@v2
+        with:
+          name: coverage-report
+          path: cov.xml
```

Comment:
Where can the resulting xml files be viewed? What tools can be used to look at them?

Reply:
See previous comment